build.py

```diff
@@ -584,14 +601,17 @@ def create_dockerfile_buildbase(ddir, dockerfile_name, argmap):
 RUN pip3 install --upgrade pip && \
     pip3 install --upgrade wheel setuptools docker
 
-# Server build requires recent version of CMake (FetchContent required)
-RUN wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | \
-      gpg --dearmor - |  \
-      tee /etc/apt/trusted.gpg.d/kitware.gpg >/dev/null && \
-    apt-add-repository 'deb https://apt.kitware.com/ubuntu/ focal main' && \
-    apt-get update && \
-    apt-get install -y --no-install-recommends \
-      cmake-data=3.18.4-0kitware1ubuntu20.04.1 cmake=3.18.4-0kitware1ubuntu20.04.1
+# Server build requires recent version of CMake (FetchContent required), build from source
```

Comment:
I think you are building from source for availability on ARM? But more recent packages are available for ARM, so I will try using 3.21.1 when I run CI

Reply:
Ah ok. I wasn't aware there was an arm build available for later cmake versions, if that's the case, then avoiding the source build is preferred.

Reply:
Revert this back to the original code and change the cmake version to 3.21.1